src/governance/governance.cpp

```diff
@@ -1281,6 +1066,11 @@ void CGovernanceManager::RequestGovernanceObject(CNode* pfrom, const uint256& nH
     connman.PushMessage(pfrom, msgMaker.Make(NetMsgType::MNGOVERNANCESYNC, nHash, filter));
 }
 
+void CGovernanceManager::AddInvalidVote(const CGovernanceVote& vote)
+{
+    cmapInvalidVotes.Insert(vote.GetHash(), vote);
```

Comment:
note: it appears we never use this as a map; only as a set. We should consider refactoring this.